src/custom/state/price/reducer.ts

```diff
@@ -137,6 +142,9 @@ export default createReducer(initialState, (builder) =>
       // Stop the loader
       state.loading = false
 
+      // Mark that the first quote is loaded
+      state.initialQuoteLoaded = true
```

Comment:
I don't think we need it

Reply:
I think its useful to have this in the state and that we have this value available anywhere in the app.